demisto/client.py

```diff
@@ -86,8 +88,78 @@ def CreateIncident(self, inc_name, inc_type, inc_severity, inc_owner, inc_labels
         return self.req("POST", "incident", data)
 
     def SearchIncidents(self, page, size, query):
-        data = {'filter': {'page': page, 'size': size, 'query': query, 'sort': [{'field':'id', 'asc': False}]}}
+        data = {'filter': {'page': page, 'size': size,
+                           'query': query, 'sort': [{'field': 'id', 'asc': False}]}}
         r = self.req("POST", "incidents/search", data)
         if r.status_code != 200:
-            raise RuntimeError('Error searching incidents - %d (%s)' % (r.status_code, r.reason))
+            raise RuntimeError(
+                'Error searching incidents - %d (%s)' % (r.status_code, r.reason))
+        return r.json()
+
+    def SearchAutomation(self, query):
+        data = {'query':query}
+        r = self.req("POST", "automation/search", data)
+        r.raise_for_status()
+        rj = r.json()
+        if "scripts" not in rj:
+            raise RuntimeError("Error searching automations - scripts attribute is not found in response")
+        return rj
+
+    def LoadAutomation(self, automation_id):
+        data = {}
+        r = self.req("POST", "automation/load/{}".format(automation_id), data)
+        r.raise_for_status()
+        return r.json()
+    
+    def SaveAutomation(self, script, filter={"query":""}, save_password=False):
+        filter["query"]="name:{}".format(script["name"])
+        data = {
+            'filter': filter,
+            'script': script,
+            'savePassword': save_password
+        }
+        r = self.req("POST", "automation", data)
+        r.raise_for_status()
         return r.json()
+
+    def DeleteAutomation(self, script, filter={"query":""}):
+        data = {
+            'filter': filter,
```

Comment:
`filter`...?

Reply:
avoided using built-in function name for variable.(7eb3498)